types/diff-match-patch/index.d.ts

```diff
@@ -40,13 +45,22 @@ export declare class diff_match_patch {
     diff_toDelta(diffs: Diff[]): string;
     diff_fromDelta(text1: string, delta: string): Diff[];
 
-    patch_make(text1: any, text2?: string): Patch[];
-    patch_deepCopy(patches: Patch[]): Patch[];
-    patch_apply(patches: Patch[], text: string): [string, boolean[]];
-    patch_fromText(text: string): Patch[];
-    patch_toText(patches: Patch[]): string;
+	match_main(text: string, pattern: string, loc: number): number;
+	match_bitap_(text: string, pattern: string, loc: number): number;
+	match_alphabet_(pattern: string): {[char: string]: number};
+
+	patch_addContext_(patch: patch_obj, text: string): void;
```

Comment:
Am I crazy or is there a mix of tabs and spaces here? Can this be fixed?

Reply:
Ah, I'll check it.

Reply:
Yeah, the formatting looks off here.